### PR TITLE
Failing tests for `{{partial` issues in 2.15.

### PR DESCRIPTION
**Pull request by rwjblue**
Related to https://github.com/emberjs/ember.js/issues/15621.

### CODE REVIEW

**Comment by GavinJoyce** [comment]
https://github.com/emberjs/ember.js/pull/15777/commits/622de5bcca119c9efc06ea4abfdd35cce604ff71 fixes three of the test failures

File: packages/ember-glimmer/tests/integration/helpers/partial-test.js [hunk 1]
```diff
@@ -73,6 +74,110 @@ moduleFor('Helpers test: {{partial}}', class extends RenderingTest {
...
+
+    this.assertText('1: 1 |2: 2 |3: 3 |');
+
+    this.runTask(() => this.context.items.pushObject({ id: 4 }));
+
+    this.assertText('1: 1 |2: 2 |3: 3 |4: 4 |');
+
+    this.runTask(() => set(this.context, 'items', {
+        items: emberA([{ id: 1 }, { id: 2 }, { id: 3 }]),
+    }));
```
Comment (GavinJoyce) [inline packages/ember-glimmer/tests/integration/helpers/partial-test.js]: This should be:

```
this.runTask(() => set(this.context, 'items',
  emberA([{ id: 1 }, { id: 2 }, { id: 3 }])
));
```

(updated here: https://github.com/emberjs/ember.js/pull/15784)

File: packages/ember-glimmer/tests/integration/helpers/partial-test.js [hunk 2]
```diff
@@ -73,6 +74,110 @@ moduleFor('Helpers test: {{partial}}', class extends RenderingTest {
...
+
+    this.assertText('apple: apple |orange: orange |banana: banana |');
+
+    this.runTask(() => this.context.items.pushObject('strawberry'));
+
+    this.assertText('apple: apple |orange: orange |banana: banana |strawberry: strawberry |');
+
+    this.runTask(() => set(this.context, 'items', {
+      items: emberA(['apple', 'orange', 'banana']),
+    }));
```
Comment (GavinJoyce) [inline packages/ember-glimmer/tests/integration/helpers/partial-test.js]: This should be:

```
this.runTask(() => set(this.context, 'items',
  emberA([{ id: 1 }, { id: 2 }, { id: 3 }])
));
```

(updated here: https://github.com/emberjs/ember.js/pull/15784)

File: packages/ember-glimmer/tests/integration/helpers/partial-test.js [hunk 3]
```diff
@@ -73,6 +74,110 @@ moduleFor('Helpers test: {{partial}}', class extends RenderingTest {
...
+    this.render(strip`
+      {{#each items as |item|}}
+        {{item}}: {{partial 'show-item'}} |
+      {{/each}}`, {
+        items: emberA(['apple', null, 'orange', 'banana']),
+      });
+
+    this.assertStableRerender();
+
+    this.assertText('apple: apple |orange: orange |banana: banana |');
```
Comment (GavinJoyce) [inline packages/ember-glimmer/tests/integration/helpers/partial-test.js]: as we have a `null` item, this should be:

```
this.assertText('apple: apple |:  |orange: orange |banana: banana |');
```

(updated here: https://github.com/emberjs/ember.js/pull/15784)

**Comment by GavinJoyce** [comment]
now that https://github.com/emberjs/ember.js/pull/15784 has landed, this can be closed

**Comment by rwjblue** [comment]
Fixed by @GavinJoyce in #15784

**Pull request closed without merging**